### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Review by OlehShumov — APPROVED** [review]

File: taxi/tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            username="testuser",
+            password="testpassword",
+            first_name="Test_first",
+            last_name="test_last",
+            license_number="123456",
+        )
+
+    def test_driver_display(self):
+        url = reverse("admin:taxi_driver_changelist")
+        res = self.client.get(url)
```
Comment (LiudmylaKulinchenko) [inline taxi/tests/test_admin.py:23]: use descriptive names, change res to result or smth

File: taxi/tests/test_form.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        self.assertFalse(form2.is_valid())
+
+    def test_driver_wrong_len_license(self):
+        form_data = {
+            "username": "test1user",
+            "password1": "test1pass",
+            "password2": "test1pass",
+            "license_number": "ABC12345678",
+            "first_name": "first1",
+            "last_name": "last1",
```
Comment (LiudmylaKulinchenko) [inline taxi/tests/test_form.py:87]: try to avoid code duplicating

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: taxi/tests/test_form.py (hunk 2 above)
Comment (Daniil-Pankieiev) [reply taxi/tests/test_form.py]: Is it okay to use this way?

**Review by LiudmylaKulinchenko — APPROVED** [review]
Great 🔥

File: taxi/tests/test_form.py (hunk 2 above)
Comment (LiudmylaKulinchenko) [reply taxi/tests/test_form.py]: Yes, looks good

**Pull request closed without merging**